cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Consider registering the `Movie` model with the admin site to manage it through the Django admin interface. You can do this by importing the `Movie` model and using `admin.site.register(Movie)`.